tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "target": "es2016",
```

Comment:
like discussed in the contributor call, would be cool to get this to a newer es version